HomePage.cs

```diff
@@ -18,28 +18,25 @@ public void startBrowser()
         {
             driver = new ChromeDriver();
             driver.Manage().Window.Maximize();
+            driver.Url = "https://www.saucedemo.com/";
+            System.Threading.Thread.Sleep(2000);
+            LogInPage loginPage = new LogInPage(driver);
+            loginPage.loginValidUser("standard_user", "secret_sauce");
         }
 
         [Test]
         public void TestLogin()
         {
-            driver.Url = "https://www.saucedemo.com/";
-            System.Threading.Thread.Sleep(2000);
             LogInPage loginPage = new LogInPage(driver);
-            loginPage.loginValidUser("standard_user", "secret_sauce");
```

Comment:
How do you test login if you don't call login method and don't pass credentials?